plugins/woocommerce/tests/e2e-pw/tests/merchant/create-cart-block.spec.js

```diff
@@ -10,40 +15,11 @@ test.describe( 'Transform Classic Cart To Cart Block', () => {
 	test.use( { storageState: process.env.ADMINSTATE } );
 
 	test( 'can transform classic cart to cart block', async ( { page } ) => {
-		// go to create a new page
-		await page.goto( 'wp-admin/post-new.php?post_type=page' );
+		await goToPageEditor( { page } );
```

Comment:
I'm not usually a fan of writing a helper to replace `page.goto`, I think it really reduces the readability of tests -- but it looks like we're also dismissing the welcome modal with this one... so I'm okay with it.